client/src/components/game/gui/GUI.scss

```diff
@@ -38,7 +38,7 @@
   .top-left-corner-cont {
     position: absolute;
     display: block;
-    top: 10px;
+    top: 1vh;
```

Comment:
What was this changed for?

Reply:
Oh forgot to revert it back to 10px when I was trying to position chat panel relative to buttons.